src/pages/messaging/agent-chat-api/index.mdx

```diff
@@ -1583,6 +1583,10 @@ https://api.livechatinc.com/v3.4/agent/action/delete_event_properties \
 
 ### Tag Thread
 
+Tags a given **thread**. 
+
+[Read how to list all available chats](https://my.livechatinc.com/settings/tags) and [create a new tag](/management/configuration-api/v2.0#add-a-tag). **Tags** are case sensitive in this method.
```

Comment:
'create a new tag' link is not linking to the appropriate part...I have checked it and it is correct, so I do not why it is happening :/